src/ui/public/filter_bar/filter_bar.js

```diff
@@ -46,10 +45,6 @@ module.directive('filterBar', function (Private, Promise, getAppState) {
 
       $scope.state = getAppState();
 
-      // Don't show filter "pinnability" when in embedded mode, as it doesn't make sense in that context
-      // as there will be no cross app navigation for which the filter should persist.
-      $scope.showFilterPin = () => chrome.getVisible();
```

Comment:
Can you explain the context behind these changes? At first glance they don't seem related to the other changes in this PR.